tests/e2e/report-submission.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { test, expect, chromium } from "@playwright/test"
+import login from "./utils/login"
+
+test.describe("Authenticated Dispatch App", () => {
+  test.beforeEach(async ({ page }) => {
+    await login(page)
+  })
+  test("Should allow me to report an incident", async ({ page }) => {
+    /* The ability to report an incident is one of the most critical
+      user stories in the Dispatch application. */
+
+    const project = "Test"
+    const type = "Customer Data"
+    const priority = "Low"
+
+    await page.getByRole("link", { name: "Report Incident" }).click()
+
+    // Title
+    await page.getByLabel("Title").click()
+    await page.getByLabel("Title").fill("Incident Test Created by Playwright")
+
+    // Description
+    await page.getByLabel("Description").click()
+    await page.getByLabel("Description").fill("test")
+
+    // Project
+    await page.getByRole("combobox").filter({ hasText: "Project" }).locator("i").click()
+    await page.getByText(project, { exact: true }).click()
+
+    // Type
+    await page.getByRole("button", { name: "Type" }).click()
+    await page.getByText(type, { exact: true }).click()
+
+    // Priority
+    await page.getByRole("button", { name: "Priority" }).click()
+    await page.getByText(priority).click()
```

Comment:
Shouldn't these be of type combobox or are you trying to mock clicking on it and loading a value?

Reply:
This clicks the button and types a value.

Reply:
It could be done either way though like you said.